recipe/meta.yaml

```diff
@@ -20,12 +20,12 @@ requirements:
     - python
     - pip
     - ruamel
-    - setuptools >=20.6.8
+    - setuptools
     - wheel
   run:
     - python
     - setuptools
-    - ruamel.yaml.clib >=0.1.2
+    - ruamel.yaml.clib >=0.2.6
```

Comment:
You are correct, that's what it says, but importing `ruamel.yaml` will fail without it in the test. Since importing is such a fundamental task, I would keep it in the run dependency. I added a comment to make this clear.